### PR TITLE
add pointers to demo location at demos README.md

### PR DESCRIPTION
**Pull request by vladimir-dudnik**
The idea is to add the paths to demos (for both distribution package and repo) to the title page.

### CODE REVIEW

File: demos/README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Open Model Zoo Demos
 
-The Open Model Zoo demo applications are console applications that demonstrate how you can use the Inference Engine in your applications to solve specific use-cases
+The Open Model Zoo demo applications are console applications that demonstrate how you can use the Inference Engine in your applications to solve specific use-cases.
+
+For the Intel® Distribution of OpenVINO™ toolkit, the demos are available after installation in the folder: `<INSTALL_DIR>\deployment_tools\open_model_zoo\demos`.
```
Comment (IRDonch) [inline demos/README.md:5]:
```suggestion
For the Intel® Distribution of OpenVINO™ toolkit, the demos are available after installation in the following directory: `<INSTALL_DIR>/deployment_tools/open_model_zoo/demos`.
```

File: demos/README.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+For the open source version of OpenVINO™ toolkit, the demos are available in the Open Model Zoo repository in the `demos` directory.
```
Comment (IRDonch) [inline demos/README.md:6]: I'm not sure this sentence makes sense. What do you mean by the open source version of the toolkit?

File: demos/README.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
+C++, C++ GAPI and Python versions are located in the `cpp`, `cpp_gapi` and `python` subfolders respectively.
```
Comment (IRDonch) [inline demos/README.md:7]:
```suggestion
C++, C++ G-API and Python versions are located in the `cpp`, `cpp_gapi` and `python` subdirectories respectively.
```

**Review by IRDonch — CHANGES_REQUESTED** [review]

File: demos/README.md (hunk 2 above)
Comment (vladimir-dudnik) [reply demos/README.md:6]: I think it reflect the version that you may build ourselves from github dldt repo

Comment (zcayetan) [reply demos/README.md:6]: Yes, open sourced is the repository on GitHub, which provides the source code
https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos

Demos folder in the Intel distribution contains what you need to run the demo when getting started

**Review by zcayetan — APPROVED** [review]

File: demos/README.md (hunk 2 above)
Comment (IRDonch) [reply demos/README.md:6]: Okay, but this sentence still doesn't sit right with me. The demos are available in the OMZ repository regardless of which version of OpenVINO toolkit is used. Also, saying "the Open Model Zoo repository" doesn't help those who don't know where this repository is, while those who _do_ know don't need any help (I imagine the conclusion that demos are in the `demos` directory is pretty obvious).

Maybe something like "The demos can also be obtained from the [Open Model Zoo repository](https://github.com/openvinotoolkit/open_model_zoo/)." would work better.

Comment (vladimir-dudnik) [reply demos/README.md]: ok, changed

**Review by IRDonch — APPROVED** [review]

File: demos/README.md [hunk 4]
```diff
@@ -1,6 +1,11 @@
 # Open Model Zoo Demos
 
-The Open Model Zoo demo applications are console applications that demonstrate how you can use the Inference Engine in your applications to solve specific use-cases
+The Open Model Zoo demo applications are console applications that demonstrate how you can use the Inference Engine in your applications to solve specific use-cases.
```
Comment (andrew-zaytsev) [inline demos/README.md:3]: Can we add a more meaningful definition here?

The Open Model Zoo demo applications are console applications that provide robust application templates to help you implement specific deep learning scenarios. These applications involve increasingly complex processing pipelines that gather analysis data from several models that run inference simultaneously, such as detecting a person in a video stream along with detecting the person's physical attributes, such as age, gender, and emotional state

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

File: demos/README.md (hunk 4 above)
Comment (vladimir-dudnik) [reply demos/README.md]: ok, changed

**Review by andrew-zaytsev — APPROVED** [review]

**Pull request merged**